.github/workflows/prebuild.yml

```diff
@@ -32,9 +32,7 @@ jobs:
           submodules: recursive
       - run: npm install
       # Supported Node versions: https://nodejs.org/en/about/releases/
-      - run: npm run prebuild -- -t 8.0.0 -t 10.0.0 -t 12.0.0 -t 14.0.0 -a ${{ matrix.config.arch }}
-      # Supported Electron versions: https://electronjs.org/docs/tutorial/electron-timelines
-      - run: npm run prebuild -- -t 4.0.4 -t 5.0.0 -t 6.0.0 -t 7.0.0 -t 8.0.0 -t 9.0.0 -t 10.0.0 -t 11.0.0 -t 12.0.0-nightly.20201013 -r electron -a ${{ matrix.config.arch }}
+      - run: npm run prebuild -- -a ${{ matrix.config.arch }}
```

Comment:
To set it up to build N-API, should this read:

```
- run: npm run prebuild -- -t 4 -r napi -a ${{ matrix.config.arch }}
```

The default is `node`

Reply:
`--runtime napi -t 4` is set in the prebuild script in `package.json`